frontend/src/_ui/Breadcrumbs/index.jsx

```diff
@@ -26,7 +27,7 @@ export const Breadcrumbs = () => {
 // define some custom breadcrumbs for certain routes (optional)
 const routes = [
   { path: '/:worspace_id', breadcrumb: 'Apps' },
```

Comment:
Hello, @arpitnath. Should we fix the typo here? form `worspace` to  `workspace`? 🤔

Reply:
Yes